vault/QAOA.md

```diff
@@ -0,0 +1,17 @@
+#quantum-information #method
+
+Double acronym: quantum approximate optimization ansatz, or quantum alternating operator ansatz. 
```

Comment:
I have also seen 'algorithm' instead of 'ansatz', i.e. Quantum Approximate Optimization Algorithm. Or is the algorithm all about choosing the ansatz only?

Reply:
Maybe algorithm is indeed the better word. I took that one from the aQa glossary, but I will ask @Zakuta at some point to write more about this. Feel free to adapt it to algorithm.

Reply:
Ok. I'll change this and then merge